packages/shared/src/components/Feed.tsx

```diff
@@ -417,6 +417,7 @@ export default function Feed<T>({
             onCommentClick={onCommentClick}
             onAdClick={onAdClick}
             onReadArticleClick={onReadArticleClick}
+            isVoting={isVoting}
```

Comment:
Do we have to do this and expose `isVoting`? Maybe as alternative approach we can just early return from mutation inside `useVote` without any user interactio. I would say spamming is not a common action so it is fine that it has no UI feedback. Also having disabled feedback kinda beats the purpose of optimistic UI?

This would also avoid a bunch of changes all over codebase.

Reply:
That makes sense. Will check this in a moment.